tests/src/Interop/StringMarshalling/LPSTR/LPSTRTestNative.cpp

```diff
@@ -126,13 +126,13 @@ extern "C" DLL_EXPORT int __cdecl Writeline(char * pFormat, int i, char c, doubl
 typedef LPCTSTR (__stdcall * Test_DelMarshal_InOut)(/*[in]*/ LPCSTR s);
 extern "C" DLL_EXPORT BOOL __cdecl RPinvoke_DelMarshal_InOut(Test_DelMarshal_InOut d, /*[in]*/ LPCSTR s)
 {
-    LPCTSTR str = d(s);
-    const char *ret = "Return";    
+    LPCWSTR str = (LPCWSTR) d(s);
```

Comment:
Don't do the cast. Instead, change `Test_DelMarshal_InOut` to return LPCWSTR instead.